core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/connection/DatabaseConnectionService.java

```diff
@@ -58,6 +58,7 @@ public CachedRowSet executeQuery(T databaseConnection, String query) throws SQLE
     public CachedRowSet executeQuery(T databaseConnection, String query, Connection connection) throws SQLException {
```

Comment:
Also add this code snippet to 'public CachedRowSet executeQueryLimitRows(T databaseConnection, String query, int limit, Connection connection) throws SQLException' (line 78)